docs/build-logs.md

```diff
@@ -69,6 +69,12 @@ Please use the "Validate Elasticsearch configuration" to verify the setup.
    alt="Configuration - Storing Jenkins Pipeline Logs in Elastic visualizing logs in Kibana and through Jenkins"
    src="./images/jenkins-config-elastic-logs-with-visualization-through-jenkins.png" />
 
+#### Log Exporter Configuration
+
+* To export logs to an OTLP endpoint, configure the following environment variables:
+- `otel.logs.exporter=otlp` : Specifies that logs should be exported using the OTLP exporter.
```

Comment:
It is configured along with the backend configuration, so it is not necessary to do it manually